src/helix-ui/elements/HXCheckboxElement.js

```diff
@@ -63,16 +65,9 @@ export class HXCheckboxElement extends HXElement {
             case 'checked':
                 if (!this.indeterminate) {
                     this.setAttribute('aria-checked', hasValue);
-                }
-                break;
-
-            case 'disabled':
-                this.setAttribute('aria-disabled', hasValue);
-                if (hasValue) {
-                    this.removeAttribute('tabindex');
-                    this.blur();
-                } else {
-                    this.setAttribute('tabindex', '0');
+                    this.$emit('change', {
+                        checked: this.checked,
+                    });
```

Comment:
Change event moved to `attributeChangedCallback` so that the event is fired either when the attribute changes or the property changes.